docs/source/introduction.rst

```diff
@@ -9,41 +9,34 @@ to another with minimal amount of downtime. This is accomplished by tailing
 and replaying the binlog while the data is being copied by a separate process.
 
 Ghostferry is a library rather than an application. The decision to make it so
-is because Ghostferry has the capability to selectively filter data to copy
-[#fselective]_. The filtering could be arbitrarily complex and thus cannot be
-easily expressed in some configuration file.
+is because Ghostferry has the capability to selectively filter data to copy.
+The filtering could be arbitrarily complex and thus cannot be easily expressed
+in some configuration file.
 
-That said, there is a generic tool known as ``ghostferry-copydb`` that will copy
-whole schemas from one MySQL to another without the basic database/table name
-filtering.
+That said, there is a generic tool known as ``ghostferry-copydb`` that will
+copy whole schemas from one MySQL to another with only the basic database/table
```

Comment:
"copy schemas" sounds like it will only create new tables of the same schema but not actually copy the data. I'd say "copy whole databases" or even "copy an entire database with all of it's tables".